From e82215424779f3a5b8e5ff9c58b610dd9907536c Mon Sep 17 00:00:00 2001
From: Anton Bambura <jenneron@protonmail.com>
Date: Tue, 19 Jan 2021 11:19:41 +0200
Subject: [PATCH 170/210] arm64: dts: qcom: msm8916-wingtech-wt86518: add
 accelerometer

This device has Kionix KXCJ9 accelerometer. Add support for it.

Signed-off-by: Anton Bambura <jenneron@protonmail.com>
---
 .../boot/dts/qcom/msm8916-wingtech-wt86518.dts     | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/msm8916-wingtech-wt86518.dts b/arch/arm64/boot/dts/qcom/msm8916-wingtech-wt86518.dts
index 110cadadd72..b010c24c2ae 100644
--- a/arch/arm64/boot/dts/qcom/msm8916-wingtech-wt86518.dts
+++ b/arch/arm64/boot/dts/qcom/msm8916-wingtech-wt86518.dts
@@ -14,6 +14,20 @@ &pm8916_usbin {
 	status = "okay";
 };
 
+&blsp_i2c2 {
+	accelerometer@e {
+		compatible = "kionix,kxcj91008";
+		reg = <0xe>;
+
+		vdd-supply = <&pm8916_l6>;
+		vddio-supply = <&pm8916_l6>;
+
+		mount-matrix = "0", "-1", "0",
+			       "-1", "0", "0",
+			       "0",  "0", "1";
+	};
+};
+
 &reg_lcd_neg {
 	regulator-min-microvolt = <5400000>;
 	regulator-max-microvolt = <5400000>;
-- 
2.34.1

